common/djangoapps/student/models.py

```diff
@@ -3086,6 +3091,67 @@ def should_celebrate_first_section(enrollment):
             return False
 
 
+class UserCelebration(TimeStampedModel):
+    """
+    Keeps track of how we've celebrated a user's progress.
+
+    An example of a celebration is a dialog that pops up after you post for the first time in the
+    discussion forums.
+
+    In general, these celebrations should be course agnostic and could show up at any point for
+    a learner. Some will be one-off celebrations (like first discussion post) while others could
+    be repeated for a user (like streaks).
+    For celebrations that can be repeated with each enrollment, see CourseEnrollmentCelebration
+
+    .. no_pii:
+    """
+    user = models.OneToOneField(User, models.CASCADE, related_name='celebrations')
```

Comment:
Maybe we can use @ciduarte as the tie-breaker. I used celebrations more because of the way it would be used, despite the 1-1 relationship. Like `user.celebrations.{specific_celebration}`
I don't have super strong feeling, but that's why celebrations makes sense to me.

Reply:
Hmm, I think I prefer celebrations... but I also don't feel strongly.

Reply:
Just to note that for the other celebration modal I made, it's singular. So this would be a delta with that. Doesn't need to be the same, but might be nice. If you like the plural, I'd say switch the other one to be plural too.